css/style.css

```diff
@@ -1,11 +1,217 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+ 
 
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+
+}
+/** navigation section
+*/
+
+.navbar {
+    display: block;
+    background-color: white;
+    position: fixed;
+    width: 100%;
+    border-bottom: 1px solid #ddd;
+}
+
+.navbar > .content {
+    height: 4.275rem;
+
+}
+
+.navbar .header-image {
+    height: 4.5rem;
+    margin-left: 30%;
+    margin-top: -9%;
+}
+
+.content {
+    margin: 0 auto;
+    padding: 0 0.7rem;
+    min-width: 37.5rem;
+}
+
+ul {
+    list-style: none;
+}
+
+ul.link a {
+    display: block;
+    padding: 8%  16%;
+    text-decoration: none;
+    text-align: center;
+    font-size: 1em;
+    float: left;
+    
+
+}
+
+
+.link {
+    list-style-type: none;
+    overflow: hidden;
+    background-color: white;
+    margin-left: 50%;
+    
 }
 
+
+.link li {
+    float: left;
+}
+
+.link li a {
+    display: block;
+    color: gray;
+    text-align: center;
+    padding: 8px 16px;
+    text-decoration: none;
+}
```

Comment:
Try to be consistent with formatting. There should be a new line after this css statement like in the rest of your code